README.rst

```diff
@@ -2,3 +2,139 @@
+changing the script accordingly:
+
+.. code-block:: console
+
+    cmsDriver.py reco -s RAW2DIGI,L1Reco,RECO,USER:EventFilter/HcalRawToDigi/hcallaserhbhehffilter2012_cff.hcallLaser2012Filter --data --conditions FT_R_53_LV5::All --eventcontent AOD --customise Configuration/DataProcessing/RecoTLR.customisePrompt --no_exec --python reco_cmsdriver2011.py
+
+
+
+3. Compute environment
+----------------------
+In order to be able to rerun the analysis even several years in the future, we
+need to "encapsulate the current compute environment", for example to freeze the
+software package versions our analysis is using. We shall achieve this by
+preparing a `Docker <https://www.docker.com/>`_ container image for our analysis
+steps.
+
+This analysis example runs within the `CMSSW <http://cms-sw.github.io/>`_
+analysis framework that was packaged for Docker in `cmsopendata
+<https://hub.docker.com/u/cmsopendata>`_. The different images corresponds to
+data sets taken in different years. Instructions can be found under
+`this repo <http://opendata.cern.ch/docs/cms-guide-docker>`_.
+
+Moreover, the re-reconstruction task needs access run-time to the condition
+database and inside a `CMS VM <http://opendata.cern.ch/search?page=1&size=20&q=virtual%20machine&subtype=VM&type=Environment&experiment=CMS>`_,
+this is achieved with the commands:
+
+.. code-block:: console
+
+    $ ln -sf /cvmfs/cms-opendata-conddb.cern.ch/FT_53_LV5_AN1_RUNA FT_53_LV5_AN1
+    $ ln -sf /cvmfs/cms-opendata-conddb.cern.ch/FT_53_LV5_AN1_RUNA.db FT_53_LV5_AN1_RUNA.db
+
+For *reana*, the condition database on CVMFS can be accessed with any
+container, the only requirement is that the user should specify the necessary
+CVMFS volumes to be live-mounted in the `reana.yaml` resource section, as
+described `here <https://reana.readthedocs.io/en/latest/userguide.html#declare-necessary-resources>`_.
+
+
+4. Analysis workflow
+--------------------
+
+First, we have to set up the environment variables accordingly for the
+`CMS SW <http://cms-sw.github.io/>`_. Although this is done in the docker
+image, `reana` overrides them and they need to be reset. This is done by
+copying the `cms entrypoint.sh script <https://github.com/clelange/cmssw-docker/blob/master/standalone/entrypoint.sh>`_:
+
+.. code-block:: console
+
+      $ source /opt/cms/cmsset_default.sh
+      $ scramv1 project CMSSW CMSSW_5_3_32
+      $ cd CMSSW_5_3_32/src
+      $ eval `scramv1 runtime -sh`
+
+
+The actual commands that are needed to carry out the analysis in the CMS
+specific environment are then:
+
+.. code-block:: console
+
+      $ ln -sf /cvmfs/cms-opendata-conddb.cern.ch/FT_53_LV5_AN1_RUNA FT_53_LV5_AN1
+      $ ln -sf /cvmfs/cms-opendata-conddb.cern.ch/FT_53_LV5_AN1_RUNA.db FT_53_LV5_AN1_RUNA.db
+      $ ls -l
+      $ ls -l /cvmfs/
+      $ cmsRun reco_cmsdriver2011.py
+
+
+5. Output results
+-----------------
+
+The demo will create ROOT files containing data in the AOD format specified
```

Comment:
Instead of "The demo will", better something like "Running the commands will..."